news/2023/2023-11-03-interview-with-ephemeral.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Interview with Ephemeral"
+date: 2023-11-03 00:00:00 +0000
+---
+
+Ephemeral, joining the game in August of 2009, has taken an active part in the development of osu!, hitting a large amount of milestones in the game. He has made one of the game's most popular beatmaps, acted as a key role in the development of the Mapping Assistance Team (MAT), and even developed many key features in the game that are still used today! Presently, he has a role of a Community Manager. A few medal hunters decided to organize an interview, which anyone could attend to ask any and all questions their souls had, as well as learn more about this well-accomplished member of this game.
```

Comment:
this feels too long to be a newspost preview line, i'd suggest making a short intro text and move this below the banner

Reply:
This was decided to just copy from 2013's interview with slight changes, but maybe these days it would be too big, I'm unsure tho

Reply:
it's really big

![image](https://github.com/ppy/osu-wiki/assets/62819481/d763ec68-6780-41e6-bdf1-2a1c91a41f18)

Reply:
Okay, yea, that's TOO big

Reply:
```suggestion
Come join us delve into the science and magnets behind medals with Ephemeral, osu!'s strongest seagull!
```

Reply:
Amazing! Only thing is there was a "to" missing

```suggestion
Come join us to delve into the science and magnets behind medals with Ephemeral, osu!'s strongest seagull!
```